pkg/extension/sumologicextension/extension.go

```diff
@@ -108,6 +123,42 @@ type OpenRegisterResponsePayload struct {
 	CollectorId            string `json:"collectorId"`
 }
 
+func (se *SumologicExtension) checkCollectorCredentials() bool {
+	if _, err := os.Stat(collectorCredentialsPath); err != nil {
```

Comment:
I believe we should account for multiple collectors being started on particular machine hence maybe hash access keys + collector name and add it the the file name?